smart_keymap/src/lib.rs

```diff
@@ -59,6 +59,8 @@ pub enum KeymapInputEventType {
 }
 
 /// Input event.
+///
+/// LIMITATION: split transport for virtual keys only works for basic key codes.
```

Comment:
The intended use for the `KeymapInputEvent` in `libsmart_keymap` is for split matrix events.

Virtual Keys are more of an implementation detail for some of the smart keys.